Assets/_BuildingBlocks/Configuration/Scripts/Configuration.cs

```diff
@@ -123,6 +126,11 @@ public void Populate(JSONNode jsonNode)
                 AllowUserSettings = jsonNode["allowUserSettings"].AsBool;
             }
 
+            if (jsonNode["corsProxyUrl"] != null)
```

Comment:
gaat dit goed? is het niet beter om string.IsNullOrEmpty te gebruiken?

Reply:
Dit gaat goed omdat `jsonNode["string"]` nil of een object van het type JsonNode teruggeeft - en niet een echte string